src/payment/strategies/googlepay/googlepay-payment-strategy.ts

```diff
@@ -102,6 +105,8 @@ export default class GooglePayPaymentStrategy implements PaymentStrategy {
 
     private _getGooglePayOptions(options: PaymentInitializeOptions): GooglePayPaymentInitializeOptions {
         if (options.methodId === 'googlepayadyenv2' && options.googlepayadyenv2) {
+            this._googlePayAdyenV2Processor?.initialize(options);
```

Comment:
Please avoid using optional chaining, what will happen if `this._googlePayAdyenV2Processor` is undefined?

You better validate is set and if not throw.